src/am_map.cpp

```diff
@@ -2583,6 +2583,50 @@ void AM_drawWalls (bool allmap)
 					}
 				}
 
+				if (line.forceAMap > 0 && (am_cheat == 0 || am_cheat >= 4)) {
+					switch (line.forceAMap)
```

Comment:
Is there any reason to use switch instead of static array with a single function call?